packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -115,15 +115,20 @@ export function trackViews(location: Location, lifeCycle: LifeCycle) {
     return { initialView, stop }
   }
 
-  function trackViewChange() {
-    return newView(lifeCycle, location, isRecording, ViewLoadingType.ROUTE_CHANGE, currentView.url)
+  function trackViewChange(startClocks?: ClocksState, name?: string) {
+    return newView(lifeCycle, location, isRecording, ViewLoadingType.ROUTE_CHANGE, currentView.url, startClocks, name)
   }
 
   return {
     addTiming: (name: string, endClocks = clocksNow()) => {
       currentView.addTiming(name, endClocks)
       currentView.triggerUpdate()
     },
+    startView: (name?: string, startClocks?: ClocksState) => {
```

Comment:
> I'm mitigated about the function name. I kind of liked newView. What made you change your mind from the RFC proposal?

A comment on the RFC about the naming.
[Mobile](https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/#custom-views) already have a `startView` method on their side and it could be more natural if we need to introduce a `stopView` at some point.


cc @BenoitZugmeyer

Reply:
Ok it makes sense! ✅